acquia/acquia_cms
# ACMS-1108: Cleanup acquia/acquia_cms module.

**Motivation**
Fixes #1148 

**Proposed changes**
Move config file from profile to acquia_cms_common module.

**Alternatives considered**
NA

**Testing steps**
- setup codebase with this branch
- require acquia_cms_common module from [this](https://github.com/acquia/acquia_cms_common/pull/17) PR 
- install site
- verify everything working fine

**Merge requirements**
